IonDotnet/Tree/IIonFloat.cs

```diff
@@ -1,7 +1,7 @@
 ﻿namespace IonDotnet.Tree
 {
-    public interface IIonFloat : IIonValue
+    public interface IIonFloat
     {
-        double Value { get; set; }
+        double FloatValue { get; set; }
```

Comment:
This method should be renamed `DoubleValue` for consistency with the other Tree interface methods that are named after their return type. We will probably add a `FloatValue` method at some point in the future that returns a `float`.

Reply:
Done